apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -0,0 +1,90 @@
+        </div>
+        <div class="metadata-form__release-date">
+          <%= Form.label(f, :issuedDate, "Release Date", class: "label label--required") %>
+          <%= Form.text_input(f, :issuedDate, value: @dataset.business.issuedDate, class: "input") %>
+        </div>
+        <div class="metadata-form__license">
+          <%= Form.label(f, :license, "License", class: "label label--required") %>
+          <%= Form.text_input(f, :license, value: @dataset.business.license, class: "input") %>
+        </div>
+        <div class="metadata-form__update-frequency">
+          <%= Form.label(f, :publishFrequency, "Update Frequency", class: "label label--required") %>
+          <%= Form.text_input(f, :publishFrequency, value: @dataset.business.publishFrequency, class: "input") %>
+        </div>
+        <div class="metadata-form__keywords">
+          <%= Form.label(f, :keywords, "Keywords", class: "label") %>
+          <%= Form.text_input(f, :keywords, value: get_keywords(@dataset), class: "input") %>
+          <div class="label label--inline">Separated by comma</div>
+        </div>
+        <div class="metadata-form__last-updated">
+          <%= Form.label(f, :modifiedDate, "Last Updated", class: "label") %>
+          <%= Form.text_input(f, :modifiedDate, value: @dataset.business.modifiedDate, class: "input") %>
+        </div>
+        <div class="metadata-form__spatial">
+          <%= Form.label(f, :spatial, "Spatial Boundaries", class: "label") %>
+          <%= Form.text_input(f, :spatial, value: @dataset.business.spatial, class: "input") %>
+        </div>
+        <div class="metadata-form__temporal">
+          <%= Form.label(f, :temporal, "Temporal Boundaries", class: "label") %>
+          <%= Form.text_input(f, :temporal, value: @dataset.business.temporal, class: "input") %>
+        </div>
+        <div class="metadata-form__organization">
+          <%= Form.label(f, :orgTitle, "Organization", class: "label label--required") %>
+          <%= Form.text_input(f, :orgTitle, value: @dataset.business.orgTitle, class: "input") %>
+        </div>
+        <div class="metadata-form__level-of-access">
+          <%= Form.label(f, :private, "Level of Access", class: "label label--required") %>
+          <%= Form.text_input(f, :private, value: get_private(@dataset), class: "input") %>
+        </div>
+        <div class="metadata-form__language">
+          <%= Form.label(f, :language, "Language", class: "label") %>
+          <%= Form.text_input(f, :language, value: @dataset.business.language, class: "input") %>
+        </div>
+        <div class="metadata-form__homepage">
+          <%= Form.label(f, :homepage, "Data Homepage URL", class: "label") %>
+          <%= Form.text_input(f, :homepage, value: @dataset.business.homepage, class: "input") %>
+        </div>
+      <% end %>
+      <%= Link.link("Cancel", to: "/", class: "btn btn--cancel metadata-form__cancel-btn") %>
+    </div
```

Comment:
```suggestion
    </div>
```
I'm guessing you want that `>`?